apps/fishing-map/features/map/map-viewport.hooks.ts

```diff
@@ -26,35 +25,45 @@ type UseViewport = {
 
 const URL_VIEWPORT_DEBOUNCED_TIME = 1000
 
-const viewportState = atom<MapCoordinates>({
-  key: 'mapViewport',
-  default: DEFAULT_VIEWPORT as MapCoordinates,
-  effects: [
-    ({ trigger, setSelf, onSet }) => {
-      const viewport = (selectViewport as any)(store.getState() as any)
```

Comment:
I had to replace recoil with [jotai](https://jotai.org/docs/core/atom) because this was not supported